src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -148,14 +148,17 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
               </div>
           </div>
           <div class="pull-right">
-            <span title="{{workItem.attributes['system.created_at'] | date:'fullDate'}}">
-              Created at: {{workItem.attributes['system.created_at'] | almMomentTime}}
+            <span *ngIf="!addNewWI" title="{{workItem.attributes['system.created_at'] | date:'fullDate'}}">
+              Created at: {{ workItem.attributes['system.created_at'] ? (workItem.attributes['system.created_at'] | almMomentTime) : 'Oops do not have the time!'}}
+            </span>
+            <span *ngIf="addNewWI" title="{{workItem.attributes['system.created_at'] | date:'fullDate'}}">
+              Creating now
             </span>
           </div>
       </div>
 
       <div class="col-md-12 col-sm-12 col-xs-12 detail-description-wrap">
-        <div class="col-md-12 col-sm-12 col-xs-12 detail-description" *ngIf="loggedIn">
+        <div class="col-md-12 col-sm-12 col-xs-12 detail-description" *ngIf="loggedIn && !addNewWI">
```

Comment:
Same logic as I explained up there. I don't think addNewWI flag is needed anywhere.